ios/FluentUI/Bottom Sheet/BottomSheetController.swift

```diff
@@ -240,6 +267,26 @@ public class BottomSheetController: UIViewController {
         expandedContentView.alpha = targetAlpha
     }
 
+    private func updateDimmingViewAlpha() {
```

Comment:
it is bit unclear to me why we need this fucntion if you are already adding alpha update in the translationanimator in line 383

Reply:
This is called during a pan to set the alpha to where it should be depending on where the sheet is right now. No animations run during the pan. The translationAnimator is only used when the user lets go of the sheet and it springs to either the expanded or collapsed state.